_pages/en_US/installing-boot9strap-(fredtool).txt

```diff
@@ -94,7 +94,7 @@ If you would prefer a visual guide to this section, one is available [here](http
 1. Use the (A) button and the D-Pad to turn on the following:
   + **"Show NAND or user string in System Settings"**
 1. Press (Start) to save and reboot
-  + If you get a black screen, [follow this troubleshooting guide](troubleshooting#black-screen-on-sysnand-boot-after-installing-boot9strap)
+  + Your device should load the Home Menu after a short delay. If you get a *consistent* black screen, [follow this troubleshooting guide](troubleshooting#black-screen-on-sysnand-boot-after-installing-boot9strap)
```

Comment:
```suggestion
  + Your device should load the Home Menu after a short delay. If you get a black screen lasting longer than 5 minutes, [follow this troubleshooting guide](troubleshooting#black-screen-on-sysnand-boot-after-installing-boot9strap)
```
5 or 10 minutes should be fine, this would alleviate some of the lumabug issues.